docs/website/netcdf-java/ncml/index.htm

```diff
@@ -53,7 +53,7 @@ <h3>Using NcML to create a NetCDF-3 file </h3>
 <ol>
   <li>Using a text or XML editor, create the NcML file with an &quot;xml&quot; or &quot;ncml&quot; file extension. Open the NcML file with <strong>NetcdfDataset.open()</strong>,
     then call <strong>ucar.nc2.FileWriter.writeFile()</strong></li>
-  <li>In the <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a> application, open an existing NcML file in
+  <li>In the <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a> application, open an existing NcML file in
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.